### PR TITLE
Update citrix-workspace from 22.03.1.32 to 22.04.0.44

### PR DESCRIPTION
**Pull request by hamlynt**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/citrix-workspace.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask "citrix-workspace" do
-  version "22.03.1.32,2203.1"
+  version "22.04.0.44,2204"
   sha256 :no_check
 
-  url "https://downloadplugins.citrix.com/Mac/CitrixWorkspaceApp.dmg"
+  url "https://downloads.citrix.com/20965/CitrixWorkspaceApp.dmg?__gda__=exp=1651072493~acl=/*~hmac=1fc66c6e1702a5b8305c829bb552fc5a6585ee68780a64d12f32ea71283c24b0"
```
Comment (bevanjkay) [inline Casks/citrix-workspace.rb:5]: @hamlynt I'm concerned that parts of the URL may be parameters that could expire and/or be version specific. Where did you find the new URL?

Comment (hamlynt) [reply Casks/citrix-workspace.rb:5]: @bevanjkay URL sourced from the Citrix [download page](https://www.citrix.com/downloads/workspace-app/mac/workspace-app-for-mac-latest.html). I agree a standard link would be preferred, the link in the previous cask points to the old version and hasn't been updated.

Comment (hamlynt) [reply Casks/citrix-workspace.rb:5]: I tested the link after stripping the parameters but it appears they are required.

Comment (hamlynt) [reply Casks/citrix-workspace.rb:5]: I also tried to add the sha256 listed on the download page, but audit fails with `Use ``sha256 :no_check`` when URL is unversioned.`

Comment (hamlynt) [reply Casks/citrix-workspace.rb]: Reverted to the original link as this has now been updated to point to the current version.

**Review by bevanjkay — APPROVED** [review]

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**